palindromic.py

```diff
@@ -1,3 +1,14 @@
-def get_longest_palindrome(s: str) -> str:
+""""Palindromic search module"""
 
-    return "x"
+
+def get_longest_palindrome(text: str) -> str:
+    """"Palindromic search function"""
+    size = len(text)
+    result = ""
+    for index in range(size):
```

Comment:
! If you start from the largest substring, the first found palindrome will be the longest